utils/db_queries.py

```diff
@@ -50,12 +52,11 @@ def server_filter(server):
         return None
 
 
-def get_zone_description(zone_id, ip_address=None, db=None):
-    if ip_address is None:
-        ip_address = cfmedb().hostname
-
-    if db is None:
-        db = Db(hostname=ip_address)
+def get_zone_description(zone_id, db=None, appliance=None):
+    if db is None and appliance is None:
+        db = Db(store.current_appliance)
```

Comment:
Shouldn't this be ```db = store.current_appliance.db```?

There is a part of me that wants to erradicate this function and move the functionality into IPAppliance, thoughts all? Make everything Appliance centric.

@mfalesni @jkrocil @quarckster